.env.test

```
@@ -0,0 +1,13 @@
+DATABASE_URL=“mysql://root:9999@127.0.0.1:3306/bossbaby_test_db”
+
+PORT = 3000
+
+TYPEORM_CONNECTION = mysql
+TYPEORM_HOST = 127.0.0.1
+TYPEORM_USERNAME = root
+TYPEORM_PASSWORD = 9999
+TYPEORM_DATABASE = bossbaby_test_db
+TYPEORM_PORT = 3306
+TYPEORM_LOGGING = TRUE
+
+SECRETKEY = "bossbaby"  
```

테스트 DB에 대한 정보도 `.env.sample` 파일에 명시해주세요!
```suggestion
// .env.sample
TYPEORM_TEST_CONNECTION = TEST_DB_TYPE
TYPEORM_TEST_HOST = TEST_DB_HOST
TYPEORM_TEST_USERNAME = TEST_DB_USER
TYPEORM_TEST_PASSWORD = TEST_DB_PASSWORD
TYPEORM_TEST_DATABASE = TEST_DB_NAME
TYPEORM_TEST_PORT = TEST_PORT
TYPEORM_TEST_LOGGING = BOOLEAN  
```